packages/manager/src/components/AccountActivation/AccountActivationLanding.tsx

```diff
@@ -49,16 +52,17 @@ const AccountActivationLanding: React.FC<CombinedProps> = (props) => {
 
   return (
     <ErrorState
+      iconClassName={classes.icon}
```

Comment:
It looks like we can use the `CustomIconStyles` prop instead of defining a new prop
```suggestion
      CustomIconStyles={{ color: '#63A701' }}
```

Reply:
🥇 Thanks for catching this. Made that change.